release-please-config.json

```diff
@@ -45,6 +45,17 @@
     },
     "packages/migrate-config": {
       "release-type": "node"
+    },
+    "packages/plugin-kit": {
+      "release-type": "node",
+      "release-as": "0.1.0",
```

Comment:
```suggestion
```

I think this isn't necessary, and we'd need to remove it after the first release so seems better not to add it.

Reply:
Actually it does seem necessary if we want to start with 0.x, I see now that new packages were released as 1.0.0.